src/slurm_plugin/clustermgtd.py

```diff
@@ -907,7 +908,69 @@ def _run_clustermgtd():
                 e,
             )
         # Manage cluster
-        cluster_manager.manage_cluster()
+        cluster_manager.manage_cluster(custom_metrics)
+        custom_metrics._has_received = True
+        sleep_remaining_loop_time(config.loop_time, start_time)
+
+
+class ClusterMetrics:
+    def __init__(self):
+        self._has_received = False
+        self._slurm_active_nodes = None
+        self._slurm_inactive_nodes = None
+        self._ec2_nodes = None
+        self._config = None
+
+    def get_config(self):
+        return self._config
+
+    def get_values(self):
+        if self._has_received:
+            return [
+                {"name": "slurm_number_active_nodes", "value": len(self._slurm_active_nodes)},
+                {"name": "slurm_number_inactive_nodes", "value": len(self._slurm_inactive_nodes)},
+                {"name": "number_ec2_nodes", "value": len(self._ec2_nodes)},
+            ]
+        return []
+
+
+def _upload_cluster_metrics(custom_metrics):
+
+    # FIXME 'StorageResolution' should not be specified I think, here just for faster tests
```

Comment:
fixme?

Reply:
Done. Removed the FIXME and StorageResolution: misunderstood its use.